spec/ruby/core/matchdata/named_captures_spec.rb

```diff
@@ -12,4 +12,14 @@
   it 'returns the latest matched capture, even if a later one that does not match exists' do
     /\A(?<a>.)(?<b>.)(?<b>.)(?<a>.)?\z/.match('012').named_captures.should == { 'a' => '0', 'b' => '2' }
   end
+
+  it 'returns a Hash with Symbol keys when symbolize_keys is provided a true value' do
```

Comment:
I believe you need to add `ruby_version_is "3.2"` or something like that - check other recently added specs.